collect_app/src/androidTest/assets/forms/field-list-repeat.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<h:html xmlns="http://www.w3.org/2002/xforms" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:jr="http://openrosa.org/javarosa" xmlns:orx="http://openrosa.org/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <h:head>
+        <h:title>Field-List Repeat</h:title>
+        <model>
+            <instance>
+                <data id="field_list_repeat">
+                    <person>
+                        <age/>
+                    </person>
+                </data>
+            </instance>
+            <bind nodeset="age" type="int"/>
+        </model>
+    </h:head>
+    <h:body>
+        <group ref="/data/person" appearance="field-list">
```

Comment:
A field-list with a single question is the same as a single question so I don't think this form adds anything. It also looks like `field-list` on a group that encloses a `repeat` doesn't have any effect. I'm not sure whether this is intentional or not. This is what I had in mind:
```xml
<?xml version="1.0"?>
<h:html xmlns="http://www.w3.org/2002/xforms" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:jr="http://openrosa.org/javarosa" xmlns:orx="http://openrosa.org/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
    <h:head>
        <h:title>Field-List Repeat</h:title>
        <model>
            <instance>
                <data id="field_list_repeat">
                    <person>
                        <name/>
                        <age/>
                    </person>
                </data>
            </instance>
            <bind nodeset="/data/person/name" type="int"/>
            <bind nodeset="/data/person/age" type="int"/>
        </model>
    </h:head>
    <h:body>
        <group ref="/data/person">
            <label>Person</label>
            <repeat nodeset="/data/person" appearance="field-list">
                <input ref="/data/person/name">
                    <label>Name</label>
                </input>
                <input ref="/data/person/age">
                    <label>What is their age?</label>
                </input>
            </repeat>
        </group>
    </h:body>
</h:html>
```

The checks would be that the + is correctly shown and that both questions are on the same screen.

Reply:
Ah right. This doesn't really feel like it has much to do with adding a repeat but more to do with just how repeats are displayed. I think it's good to have a test but I might put it in a different file.

Reply:
Scratch that. Disagree with myself. Adding repeats is basically the core of repeats are. Ignore me.

Reply:
Oh wow it actually did need the code in `FormEntryViewModel`. Who'd have thunk it.